lib/project_types/theme/commands/serve.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Theme
+  module Commands
+    class Serve < ShopifyCli::Command
+      def call(*)
+        CLI::UI::Frame.open(@ctx.message('theme.checking_themekit')) do
+          Themekit.ensure_themekit_installed(@ctx)
+        end
```

Comment:
I wonder if this is worth running in a frame. It looks a bit weird when themekit is already installed:

```
% shpfy serve
┏━━ Verifying Theme Kit ━━━━━━━━━━━━━━━━━━━━━━━━
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ (0.01s) ━━
```

Reply:
This will be changed in a later PR. For now, it'll be an empty frame, but it'll become a spinner for all commands in one PR.